setup.py

```diff
@@ -48,8 +48,10 @@ def get_long_description():
     keywords="taxii taxii2 server json cti cyber threat intelligence",
     packages=find_packages(exclude=["*.test", "*.test.data"]),
     install_requires=[
+        "environ-config",
```

Comment:
This changeset depends on a PR I have up for this package. If it doesn't get pulled then I have an alternative approach that isn't as nicely declarative but would work fine with the currently released versions of the environs package.

Reply:
This got merged into `environ-config` 21.1.0. Shall we set this `>=21.1`?